docs/index.rst

```diff
@@ -0,0 +1,52 @@
+.. starcraft documentation root file
```

Comment:
should we rename all instances of `starcraft` here to `craft-archives`? Or is that going to happen in a future change?